src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -210,7 +210,7 @@
   <!-- NFS -->
   <li routerLinkActive="active"
       class="item tc_menuitem_nfs"
-      *ngIf="permissions?.nfs?.read">
+      *ngIf="enabledFeature.nfs && permissions?.nfs?.read">
```

Comment:
```suggestion
      *ngIf="enabledFeature$.nfs && permissions?.nfs?.read">
```

Reply:
Thanks for the suggestion @votdev! The error actually is caused by the fact that feature-toggles code in Navigation component was (accidentally) removed during Boostrap upgrade (#25188). I'll need to fix that too, but given @tspmelo is working on the Navigation component right now (#31923), I'll defer this work until that's merged.